File: py/fizz-buzz.py
def fizz_buzz(n):
  ans = ''
  if n % 3 == 0:
    ans = 'Fizz'
  if n % 5 == 0:
    ans = ans + 'Buzz'
  if ans:
    return ans
  else:
    return n 
  
for n in range(1,101):
  print(fizz_buzz(n))

File: cs/FizzBuzzExtensionMethod/FizzBuzz.cs
using System;

namespace FizzBuzz
{
    public static class FizzBuzzExtension
    {
        public static string FizzBuzz(this int n)
        {
            string substitute = String.Format(
                "{0}{1}",
                n % 3 == 0 ? "Fizz" : String.Empty,
                n % 5 == 0 ? "Buzz" : String.Empty);
            return string.IsNullOrEmpty(substitute) ? n.ToString() : substitute;
        }
    }
}
